src/hooks/screen/useCreateChat.ts

```diff
@@ -21,7 +20,7 @@ const useCreateChat = () => {
   const {goToChatScreen} = useChatUtilsHook();
   const {signedProfileId, anonProfileId} = useUserAuthHook();
 
-  const handleMemberSchema = (response) => {
+  const handleMemberSchema = async (response) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleMemberSchema` function has been updated to be an async function. This is a good change as it allows for better error handling and makes the code more readable by using `await` instead of chaining promises.



<!-- This is an auto-generated comment by OSS CodeRabbit -->